### ISSUE TITLE
WANTED: Programmers/Maintainers

### ISSUE THREAD

**Issue by braph** [braph/nbfc-linux#10]
I'm looking for C-programmers who want to join this project in an github organization as maintainers and programmers.
The project itself is in a finished state, the codebase is IMHO clean.
Please write in this thread if you are interested and tell me what you are willing to do (maintaining pull requests for new configs or writing actual code)

**Comment by BachoSeven** [comment]
I'm not a programmer although I do have some experience with C, but I can help with PRs for new configurations.

As for packaging, as per #1 I'm maintaining both `nbfc-linux` and `nbfc-linux-git` in the AUR; however as this project reaches a stable state it might be worth addin it to the actual Arch repos, since it's also documented in the wiki; would have to find a [TU](https://wiki.archlinux.org/title/Trusted_Users) to maintain it for that though.

**Comment by braph** [comment]
Yeah, I edited the wiki so people at least know about the native solution for this problem.
I created the organization 'nbfc-linux' and invited you.

**Comment by braph** [comment]
@BachoSeven Can you please use the new nbfc-linux/nbfc-linux repository for your PKGBUILDs? I will remove this repository then. Thanks!

**Comment by BachoSeven** [comment]
Also updated the URL in the wiki page.

**Comment by braph** [comment]
@BachoSeven No, old issues will not be migrated.
